jiraannouncer/views/fcms.py

```diff
@@ -0,0 +1,43 @@
+import hmac
+
+from pyramid.view import view_config
+from sys import hexversion
+
+from ..utils import send, devsay
```

Comment:
devsay is not needed in this view.

Reply:
I am using it though, in case we can't decode the JSON body posted.